futures/examples/client.rs

```diff
@@ -82,7 +84,7 @@ fn main() {
             c.basic_ack(message.delivery_tag, false)
           })
         })
-      })
+      }).map_err(Error::from)
     }).map_err(|err| eprintln!("error: {:?}", err))
```

Comment:
From what I understand, we could/should drop the `:?` in places like that, right?